cmd/aries-agentd/startcmd/webhook/webhook.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webhook
```

Comment:
@DRK3 did you evaluate a 3rd party library for websockets? Eg. gorilla/websocket

Reply:
@DRK3 and I investigated this further yesterday:

ACA-Py...
* Can notify events through webhooks and websockets
   * If both webhooks and websockets are registered then all events are sent to both channels
* Only registers webhooks via the command line parameter `--webhook-url`
* Registers websockets when the client does HTTP GET of `/ws`
* Uses the websocket channel for one purpose only - to send events to the client. ACA-Py does not listen for incoming commands via this channel.

Reply:
Documentation has begun: https://hackmd.io/4MYrV-f3TYiwMKvIVgMn6A?view

I suggest watching that document and also for any PRs to the ACA-Py repo that add this docu.

Reply:
@DRK3 they've documented the webhook functionality in this PR: https://github.com/hyperledger/aries-cloudagent-python/pull/212/files

Interestingly, they do not mention the websocket functionality...